tests/FetchFunctionalTest.php

```diff
@@ -21,7 +21,7 @@ public function testItFetchesRootHtml(): void
         self::assertSame('OK', $response->status()->reasonPhrase());
         self::assertTrue($response->headers()->contains('Content-Type', 'text/html'));
         self::assertTrue($response->headers()->has('etag'));
-        self::assertSame('354', $response->headers()->get('Content-Length'));
+        self::assertSame('355', $response->headers()->get('Content-Length'));
```

Comment:
Changing this expected because adding the EOF for `tests/static/index.html` file.